src/client/components/AccordionItem/index.test.js

```diff
@@ -6,7 +6,9 @@ describe("<AccordionItem />", () => {
   it("renders the component", async () => {
     const wrapper = renderNonTransContent(Component, "AccordionItem", {
       header: <div>header react element</div>,
-      expandedBody: <div>detailed body react element</div>,
+      content: <div>detailed body react element</div>,
+      showContent: [true, true, true, true],
```

Comment:
Is this supposed to be an array? It looks like it should be a bool.

Reply:
good point, nice catch!